jekyll/archived/snippets/built-in-env-vars.md

```diff
@@ -0,0 +1,29 @@
+Variable | Type | Value
```

Comment:
I think as well as being added to the archived folder this one still needs deleting from `_includes/snippets`?